catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraPublishers.scala

```diff
@@ -23,7 +22,7 @@ trait SierraPublishers {
   // https://www.loc.gov/marc/bibliographic/bd260.html
   // https://www.loc.gov/marc/bibliographic/bd264.html
   //
-  def getPublishers(bibData: SierraBibData): List[Agent] = {
+  def getPublishers(bibData: SierraBibData): List[Organisation] = {
```

Comment:
Double-checking: because Organisation is a subclass of Agent, we could use `List[Agent]` for a list that contained Agents or Organisations?

Reply:
yes. Actually `publishers` on `Work` is a `List[Agent]`. I changed the return type of this function cause it seems like in this case we always return `Organisation`

Reply:
I don't see `Event` as subclass of `Agent` in @silveroliver model.. But definitely we can for `Agent`, `Person` and `Organisation`

Reply:
It’s a funny one, and maybe we need to rethink creators and/or organisations? The relevant bit is here:

https://github.com/wellcometrust/platform/blob/master/ontologies/WIP/sierratransformable#L75-L77

and in fact, Silver was going to look at exactly how that works, because creators has flipped from being an Agent to being a QualifiedConcept, which can in turn specialise to Organisation.

Aren't types fun? 😄

Reply:
@alicefuzier no we dont have event as agent at the moment. In theory, in MARC, you can have a meeting as the creator of a work. I dont know if a) this actually happens in practice b) we think this is sensible.

Regarding QualifiedConcept I was going to suggest we introduce a QualifiedAgent. Bit yucky but bascially what is happening here. Think need some collective whiteboard time when I am back in.

Reply:
I see... I don't think this affects this PR 'cause if creators can be both `Agent` and `Event` and there is no common supertype for both, then we're a bit screwed anyway and we have to define creators as `List[Any]' (which I'd rather not do). But yeah, maybe we should take that into account